sympy/solvers/inequalities.py

```diff
@@ -982,3 +986,482 @@ def reduce_inequalities(inequalities, symbols=[]):
 
     # restore original symbols and return
     return rv.xreplace({v: k for k, v in recast.items()})
+
+
+class UnboundedLinearProgrammingError(Exception):
+    """
+    A linear programing problem is said to be unbounded if its objective
+    function can assume arbitrarily large values.
+
+    Example
+    =======
+
+    Suppose you want to maximize
+        2x
+    subject to
+        x >= 0
+
+    There's no upper limit that 2x can take.
+    """
+    pass
+
+
+class InfeasibleLinearProgrammingError(Exception):
+    """
+    A linear programing problem is considered infeasible if its constraint set
+    is empty. That is, if the set of all vectors satisfying the contraints is
+    empty, then the problem is infeasible.
+
+    Example
+    =======
+
+    Suppose you want to maximize
+        x
+    subject to
+        x >= 10
+        x <= 9
+
+    It's not possible for x to satisfy the given contraints.
+    """
+    pass
+
+
+def _pivot(M, i, j):
+    """
+    There are four rules for the pivot operation:
+    - The pivot quantity goes into its reciprocal
```

Comment:
Does this just mean "- the pivot is not zero"?

Reply:
The pivot quantity is entree i,j. This is referring to this line:
`
MM[i,j] = 1 / Mij
`

I should probably make the order correspond to the code more and maybe use different language.